Desktop/mobile apps need branding to work with OIDC (#3492)
owncloud/docs: modules/admin_manual/pages/configuration/user/oidc/oidc.adoc

ownCloud desktop and mobile clients detect whether OIDC is available (service discovery) and use this login method when a new account is created.

> NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. Some IDP's like MS-Azure, these and other required parameters come from the IPD and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.

Reviewer: ```suggestion
NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. For some IDP's like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.
```

> NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. For some IDP's like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.

Reviewer 2: ```suggestion
NOTE: The desktop and mobile apps (clients) have a default client ID and secret hard-coded, which are used for ownCloud's oauth2 app. When using Kopano as IDP, it does not pre-define a client ID and secret. You can use the default ones of the client to configure Kopano properly. With some IDPs like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according to your needs.
```
